src/Spd.Resource.Repository/Biz/BizRepository.cs

```diff
@@ -102,13 +102,17 @@ public async Task<IEnumerable<BizResult>> QueryBizAsync(BizsQry qry, Cancellatio
 
             if (biz == null) throw new ApiException(HttpStatusCode.NotFound);
 
-            if (!IsSoleProprietor(updateBizCmd.BizType) && IsSoleProprietor(SharedMappingFuncs.GetBizTypeEnum(biz.spd_licensingbusinesstype)))
+            //TODO: updateBizCmd.UpdateSoleProprietor is it a flag for update execution?

```

Comment:
UpdateSoleProprietor is a flag. When user do login, we do not have its sole proprietor info, so, during login process, in manager layer, we need to set this flag to false.

If update is called from biz profile manager, then set this flag to true.